qiskit/pulse/library/symbolic_pulses.py

```diff
@@ -826,6 +826,151 @@ def __new__(
+
+    .. math::
+
+        \\text{risefall} &= \\text{risefall_sigma_ratio} \\times \\text{sigma}\\\\
+        \\text{width} &= \\text{duration} - 2 \\times \\text{risefall}
+
+    If ``width`` is not None and ``risefall_sigma_ratio`` is None:
+
+    .. math:: \\text{risefall} = \\frac{\\text{duration} - \\text{width}}{2}
+
+    Gaussian :math:`g(x, c, σ)` and lifted gaussian :math:`g'(x, c, σ)` curves
+    can be written as:
+
+    .. math::
+
+        g(x, c, σ) &= \\exp\\Bigl(-\\frac12 \\frac{(x - c)^2}{σ^2}\\Bigr)\\\\
+        g'(x, c, σ) &= \\frac{g(x, c, σ)-g(-1, c, σ)}{1-g(-1, c, σ)}
+
+    From these, the lifted DRAG curve :math:`d'(x, c, σ, β)` can be written as
+
+    .. math::
+
+        d'(x, c, σ, β) = g'(x, c, σ) \\times \\Bigl(1 + 1j \\times β \\times\
+            \\Bigl(-\\frac{x - c}{σ^2}\\Bigr)\\Bigr)
+
+    The lifted gaussian square drag pulse :math:`f'(x)` is defined as:
+
+    .. math::
+
+        f'(x) &= \\begin{cases}\
+            \\text{A} \\times d'(x, \\text{risefall}, \\text{sigma}, \\text{beta})\
+                & x < \\text{risefall}\\\\
+            \\text{A}\
+                & \\text{risefall} \\le x < \\text{risefall} + \\text{width}\\\\
+            \\text{A} \\times \\times d'(\
+                    x - (\\text{risefall} + \\text{width}),\
+                    \\text{risefall},\
+                    \\text{sigma},\
+                    \\text{beta}\
+                )\
+                & \\text{risefall} + \\text{width} \\le x\
+        \\end{cases}\\\\
+
+    where :math:`\\text{A} = \\text{amp} \\times
+    \\exp\\left(i\\times\\text{angle}\\right)`.
+
+    Args:
+        duration: Pulse length in terms of the sampling period `dt`.
+        amp: The amplitude of the DRAG rise and fall and of the square pulse.
+        sigma: A measure of how wide or narrow the DRAG risefall is; see the class
+               docstring for more details.
+        beta: The DRAG correction amplitude.
+        width: The duration of the embedded square pulse.
+        angle: The angle in radians of the complex phase factor uniformly
+            scaling the pulse. Default value 0.
+        risefall_sigma_ratio: The ratio of each risefall duration to sigma.
+        name: Display name for this pulse envelope.
+        limit_amplitude: If ``True``, then limit the amplitude of the
+            waveform to 1. The default is ``True`` and the amplitude is constrained to 1.
+
+    Returns:
+        SymbolicPulse instance.
+
+    Raises:
+        PulseError: When width and risefall_sigma_ratio are both empty or both non-empty.
+    """
+    # Convert risefall_sigma_ratio into width which is defined in OpenPulse spec
+    if width is None and risefall_sigma_ratio is None:
+        raise PulseError(
+            "Either the pulse width or the risefall_sigma_ratio parameter must be specified."
+        )
+    if width is not None and risefall_sigma_ratio is not None:
```

Comment:
```suggestion
    elif width is not None and risefall_sigma_ratio is not None:
```

Reply:
I don't have a strong preference for `if` vs `elif`, but I copied the format from the GaussianSquare code. I believe pylint at least has a nag to favor `if` over `elif` when following a `raise`/`return`/`break`/`continue` statement.